### ISSUE TITLE
Does the program support multiple antibodies in one complex?

### ISSUE THREAD

**Issue by bzhousd** [TencentAI4S/tfold#6]
Dear Authors,  

Thank you for your great work.  

Alphafold  is able to take many chains,  while I am not sure what format of input I should use for tfold-ag.  for example I have  chains like  H(heavy chain) L (light chain) A(antigen)  B(heavy chain), C(light chain), D( antigen).

so prot_ids.abag.txt will be
1PDB_H_L_A_B_C_D

and input msa file will have two msa in one a3m file, the order should be A and D, and separated by 0x00

Thanks

**Comment by wufandi** [comment]
Hi,
Thank you for your interest in our work. Unfortunately, tFold-Ag is currently unable to handle multi-chain antigen/multi-antibody input scenarios.
There is a method that might meet your needs: 
1. Use tFold-Ag to predict **1PDB_H_L_A**, 
2. Use tFold-Ag to predict **1PDB_B_C_D**. 
3. Use AlphaFold to predict **1PDB_A_B**. 
4. Align the three complexes together based on A and B.

**Comment by bzhousd** [comment]
Thank you for your prompt reply.  I have another question to ask.  I wonder if tfold could output the equivalent pLDDT, 

Thanks

**Comment by wufandi** [comment]
Yes, the plddt output by tFold-Ag is equivalent, but I would recommend you to judge the accuracy of the model prediction based on the ipTM score. It is highly correlated with the accuracy of the epitope.
